modules/migration-upgrading-mtc-on-ocp-3.adoc

```diff
@@ -23,14 +23,13 @@ You can upgrade {mtc-full} ({mtc-short}) on {product-title} 3 with `podman`.
 $ sudo podman login registry.redhat.io
 ----
 
-. Download the latest `operator.yml` file:
+. Download the `operator.yml` file:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ sudo podman cp $(sudo podman create \
-  registry.redhat.io/rhmtc/openshift-migration-rhel7-operator:v{mtc-version}):/operator.yml ./ <1>
+  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.5.1):/operator.yml ./
```

Comment:
Same as above comments. `operator-3.7.yml` is only for OCP v3.7 clusters, `operator.yml `is for OCP v3.9 to OCP v4.5 clusters